Gaijinizer patch/CommonEvents_119.txt

```diff
@@ -31,35 +31,38 @@ Ah……!!AAhhhh......!!#Triketos
 本当は口での搾精がメインで、そこに座れるのは
 特別な　『エサ』　だけなんだから…♡
 > CONTEXT: CommonEvents_119/24/Dialogue/enemy2
-You feel good, right?Empuse-chan's milking vagina
-... ♡. The main thing is that milking in the
-mouth is the main thing, and it is possible to
-sit there. Because it is only a special "food"
-...#MTLed
+That feels good, right? Being inside Empuse-chan's
+semen-milking vagina...♡ Her main specialty is
+milking with her mouth, only very special "food"
+gets to sit where you are... ♡#Ron
 > END STRING
```

Comment:
A quick rearrangement for flow:
>_**It feels good being inside Empuse-chan's semen
milking vagina... right? ♡ Her main specialty is
milking with her mouth, so only very special "food"
gets to sit where you are... ♡**_